terminal.go

```diff
@@ -196,18 +196,39 @@ type LightningTerminal struct {
 
 	restHandler http.Handler
 	restCancel  func()
+
+	mobileListener  *bufconn.Listener
+	mobileReadyChan chan struct{}
 }
 
 // New creates a new instance of the lightning-terminal daemon.
-func New() *LightningTerminal {
+func New(listener *bufconn.Listener, readyChan chan struct{}) *LightningTerminal {
 	return &LightningTerminal{
-		statusMgr: status.NewStatusManager(),
+		statusMgr:       status.NewStatusManager(),
+		mobileListener:  listener,
+		mobileReadyChan: readyChan,
 	}
 }
 
 // Run starts everything and then blocks until either the application is shut
 // down or a critical error happens.
 func (g *LightningTerminal) Run() error {
+	// DEBUG: log to file
+	stdoutPath := "/data/user/0/com.liminal/files/stdout.log"
+	stderrPath := "/data/user/0/com.liminal/files/stderr.log"
```

Comment:
liminal, ey?

Reply:
Working on pushing some cleanup commits to pull this from terminal Config

Reply:
Cleaned this up in the latest version.